### DISCUSSION TITLE
How long can I be signed in for on IB-Gateway?

### DISCUSSION THREAD

**Discussion by ProjectsByJackHe** [gnzsnz/ib-gateway-docker#171] [Q&A]
Hi! Thanks for this project :)

I deployed the **IB Gateway** image on an Azure Ubuntu headless server.

I was wondering how much manual maintenance there is? 

How long am I signed in for? Because of 2FA, I need to be present when the bot signs out.

**Comment by gnzsnz** [comment]
please check this [discussion](https://github.com/gnzsnz/ib-gateway-docker/discussions/145#discussioncomment-10357342)
